src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -27,13 +30,17 @@ public abstract class UbiquitousLanguageAsFile implements UbiquitousLanguageRepo
   public void saveAll(Set<UbiquitousLanguage> ubiquitousLanguages) {
     try {
       Files.createDirectories(destinationFolder);
+      Manifest manifest = new Manifest();
 
       for (UbiquitousLanguage ubiquitousLanguage : ubiquitousLanguages) {
         UbiquitousLanguageDto dto = UbiquitousLanguageDto.from(ubiquitousLanguage);
         String fileName = dto.getName().toLowerCase().replace(" ", "_");
         serializer.writeValue(destinationFolder.resolve(fileName + "." + extension).toFile(), dto);
+
+        manifest.addFile(destinationFolder.resolve(fileName + "." + extension).toString());
```

Comment:
Make a variable for the concatenation between the destination folder, the file name and its extension, in order not to do it twice.